tutorials/physics/using_jolt_physics.rst

```diff
@@ -0,0 +1,473 @@
+Instead :ref:`cast_motion<class_PhysicsDirectSpaceState3D_method_cast_motion>` is
+implemented with a binary search, similar to how Godot Physics does it.
+
+However, in Godot Physics this binary search is hardcoded to 8 steps, which tends to
+result in quite jittery output over even moderate distances. With Godot Jolt (and
+consequently this module) the number of steps is dynamically calculated  based on
+the cast distance, aiming for roughly millimeter precision, and then clamped between
+4 and 16 steps.
+
+This does however mean that :ref:`cast_motion<class_PhysicsDirectSpaceState3D_method_cast_motion>`
+will technically, when using Jolt, cost more CPU performance the farther you cast
+the shape.
+
+Note that this also applies to :ref:`body_test_motion<class_PhysicsServer3D_method_body_test_motion>`, and consequently :ref:`test_move<class_PhysicsBody3D_method_test_move>`,
+:ref:`move_and_collide<class_PhysicsBody3D_method_move_and_collide>` and :ref:`move_and_slide<class_CharacterBody3D_method_move_and_slide>`.
+
+Baumgarte stabilization
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Jolt employs a technique in its solver called Baumgarte stabilization, which is
+meant to mitigate constraint drift within the simulation, resulting in a more stable
+simulation. This technique can however result in some artifacts, like piles of
+bodies not separating as quickly as one might expect.
+
+The strength of this stabilization can be tweaked using the project setting
+:ref:`physics/jolt_physics_3d/simulation/baumgarte_stabilization_factor<class_ProjectSettings_property_physics/jolt_physics_3d/simulation/baumgarte_stabilization_factor>`.
+Setting this project setting to ``1.0`` will effectively disable the technique,
+resulting in a simulation that more closely resembles Godot Physics, but which is
+also more unstable.
+
+Motion queries (move_and_slide, etc.)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Physics servers in Godot are meant to implement the :ref:`PhysicsServer3D.body_test_motion()<class_PhysicsServer3D_method_body_test_motion>`
+method, which in turn powers methods like :ref:`PhysicsBody3D.test_move()<class_PhysicsBody3D_method_test_move>`,
+:ref:`PhysicsBody3D.move_and_collide()<class_PhysicsBody3D_method_move_and_collide>`
+and :ref:`CharacterBody3D.move_and_slide()<class_CharacterBody3D_method_move_and_slide>`,
+which are largely meant to be used for moving player characters around.
+
+:ref:`PhysicsServer3D.body_test_motion<class_PhysicsServer3D_method_body_test_motion>`
+is split into three parts, the first being depenetration (called "recovery" in
+Godot), the second being a shape-cast from the "recovered" position, and the third
+being the actual collision check, at the "unsafe" fraction of the shape-cast.
+
+The "recovery" step in Godot Physics is hardcoded to always do 4 iterations with 40%
+depenetration per iteration. I figured these constants might be useful to expose, so
```

Comment:
'I figured' is probably not what you want in the documentation.

Reply:
See [my comment](https://github.com/godotengine/godot-docs/pull/10526#issuecomment-2664160981) below. This entire section should be removed in my opinion.